### PR TITLE
support histogram

### PR DESCRIPTION
**Pull request by mtanda**
client_ruby support histogram.
https://github.com/prometheus/client_ruby/pull/24

I want to add histogram metric type to this plugin.

### CODE REVIEW

**Comment by kazegusuri** [comment]
nice work! first look is good. I will try it tomorrow.

**Comment by mtanda** [comment]
Thanks lot!!

I think this test code is not good...
https://github.com/kazegusuri/fluent-plugin-prometheus/pull/8/files#diff-8fc5cc35e833a49ca2fbebc4c3266dc4R153

I'm not familiar with RSpec and fluentd plugin testing...
If there is any other good way, I'll fix it.

**Comment by kazegusuri** [comment]
I fixed test failure at master branch. Please merge the fix, then your test will pass.


> I think this test code is not good...
> https://github.com/kazegusuri/fluent-plugin-prometheus/pull/8/files#diff-8fc5cc35e833a49ca2fbebc4c3266dc4R153

Not so bad. There can be improved, but acceptable.

File: lib/fluent/plugin/prometheus.rb [hunk 1]
```diff
@@ -192,5 +194,33 @@ def instrument(record, expander, placeholders)
...
+    class Histogram < Metric
+      def initialize(element, registry, labels)
+        super
+        if @key.nil?
+          raise ConfigError, "histogram metric requires 'key' option"
+        end
+
+        begin
+          if element['buckets']
+            buckets = element['buckets'].split(/,/).map do |e|
```
Comment (kazegusuri) [inline lib/fluent/plugin/prometheus.rb]: The split makes space prefixed numbers such as " 0.10", which results in 0 by `to_i`. You need to call `map(&:strip)` after split.

Comment (mtanda) [inline lib/fluent/plugin/prometheus.rb]: Sorry, I don't check it enough. I fixed it.

**Comment by mtanda** [comment]
I rebase branch and fix one bug. The test is passed.
Please check :-)

**Comment by kazegusuri** [comment]
Thank you for contributing. I have released v0.2.0.

**Comment by mtanda** [comment]
Thanks!! 😄

**Pull request merged**